modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -29,8 +29,116 @@ payload
 
 // CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// Flat File config properties are shared with COBOL Copybook and Fixed Width.
-include::partial$dataweave-formats-flatfile-config.adoc[]
+
```

Comment:
TODO: compare with the include and remember to update with new property in Jan v2.4 patch update.